Tchap/Modules/Application/AppCoordinatorType.swift

```diff
@@ -27,6 +27,12 @@ protocol AppCoordinatorType: Coordinator {
     /// - Returns: true to indicate that the activity has been handled, or false to let iOS handle the activity.
     func handleUserActivity(_ userActivity: NSUserActivity, application: UIApplication) -> Bool
     
+    /// Resume the application by selecting a room.
+    ///
+    /// - Parameters:
+    ///   - roomId: the room identifier.
+    func resumeBySelectingRoom(with roomId: String)
```

Comment:
Maybe you could simply add a method  `func  userDidTapNotification(with userInfo: [String: Any])` and handle the action to make in `AppCoordinator`